Scripts/Services/VeteranRewards/StatRewardGump.cs

```diff
@@ -7,40 +7,31 @@ namespace Server.Engines.VeteranRewards
     public class StatRewardGump : BaseGump
     {
         public StatRewardGump(PlayerMobile from)
-            : base(from, 50, 50)
+            : base(from, 200, 200)
         {
             from.CloseGump(typeof(StatRewardGump));
+            TypeID = 0x193;
         }
 
         public override void AddGumpLayout()
         {
-            AddBackground(0, 0, 291, 173, 9200);
-            AddImageTiled(5, 5, 281, 140, 2624);
-
-            //AddHtmlLocalized(7, 7, 277, 126, 1076664, 0x7FFF, false, false);
-            /*<B>Ultima Online Rewards Program</B><BR>Thank you for being part of the Ultima Online community for over 6 months. 
-             * As a token of our appreciation, your stat cap will be increased. */
```

Comment:
ServUO and RunUO use 1 month = 1 year for vet rewards. This is why the change was made to teh stat rewards in the first place.